### ISSUE TITLE
[SUGGESTION] gitter.im room

### ISSUE THREAD

**Issue by xQuByte** [brunnopleffken/addictive-community#96]
I propose to create root in gitter.im. It's a good way to communicate :)

**Comment by xQuByte** [comment]
@brunnopleffken So.. ? :)

**Comment by brunnopleffken** [comment]
Sounds nice, and I liked the integration with Git.
But what's the possibility of seeing everybody actively participating?

**Comment by xQuByte** [comment]
@brunnopleffken A person who has joined the room.

**Comment by xQuByte** [comment]
@brunnopleffken And the person who writes the chat :)

**Comment by halojoy** [comment]
I think it is better to start a version of addictive community forum.
Such a forum could be a good way to see if there are any bugs.
Using a forum can be very good.
I suggest installing v0.10.0 and add link to this forum.

**Comment by xQuByte** [comment]
@halojoy Gitter is to communicate to developers and "developer friends"

**Comment by johnforte** [comment]
I agree with @halojoy, why use something else to manage all discussion when you can use the thing you built!

**Comment by xQuByte** [comment]
@johnforte Because Gitter is a chat, not forum.

**Comment by brunnopleffken** [comment]
I've been thinking about it in recent days, and I still don't see reason to split the discussion between here and there. Here at GitHub we had several moments where ideas were encouraged and taken into account.

I fear that the division between two channels can reduce the movement of ideas here; or cause abandonment of Gitter in a few days. If there's an idea that I liked and I can take into account the future is to create a forum using our own product: Addictive Community.

Maybe we can bring back to the fore the idea of Gitter when the activity started to be high! :)
